### DISCUSSION TITLE
Enabling linear advanced S1

### DISCUSSION THREAD

**Discussion by Ar1sd** [mriscoc/Ender3V2S1#1305] [Q&A]
I have an Ender 3 S1 and I have been using this firmware for about a day now and enjoyed it a lot. I want to enable linear advanced for the printer. However, I can't find the non-compiled code for the ender 3 s1 only for the v2.

**Comment by mriscoc** [comment]
For know how to compile a custom version look at: https://github.com/mriscoc/Special_Configurations#creating-your-own-custom-configuration
